_submissions/round-12/1/2015-02-22-plotting-function-R.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2015-02-22
+round: Round 12
+title: Second Meeting for Round 12
+author: Ben Weinstein
+tags:
+  - R
+  - plot
+---
+
+#Concept Map for Plot() in R
```

Comment:
Please move this to the `title` field in the header - it will automatically be used in the body of the page.